### DISCUSSION TITLE
Does viem's integration with multicall support batching multiple writes into a single transaction?

### DISCUSSION THREAD

**Discussion by bam4564** [wevm/viem#1095] [Question]
Looking at the docs, it's a little unclear to me if viem supports multicall for writes. [Here](https://viem.sh/docs/contract/multicall.html#multicall) is the relevant section of the docs. 

I looked at the [multicall3](https://github.com/mds1/multicall) docs and they do support writes.

**Comment by tmm** [comment]
https://github.com/wagmi-dev/viem/discussions/741#discussioncomment-6227292
